<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEditForm</class>
 <widget class="QWidget" name="UserEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>201</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>User Edit</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../assets/logo.svg</normaloff>../../assets/logo.svg</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Prefix Name : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrefixName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>First Name : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Last Name : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLastName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Email : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPhone">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Phone : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddress">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Address : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditPrefixName</tabstop>
  <tabstop>lineEditFirstName</tabstop>
  <tabstop>lineEditLastName</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>lineEditAddress</tabstop>
  <tabstop>pushButtonSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
